build-system/global-configs/OWNERS

```diff
@@ -15,5 +15,9 @@
       pattern: 'client-side-experiments-config.json',
       owners: [{name: 'danielrozenberg'}, {name: 'jridgewell'}],
     },
+    {
+      pattern: '*-config.json',
+      owners: [{name: 'alanorozco'}],
```

Comment:
I think this is too narrow since I believe this will funnel all automatic review requests from owners bot to you.

What if we list the WGs who generally need to modify this file? Or expand it to `ampproject/working-groups`? Or alternatively, list a subset with `requestReviews: false` so the parent directory's owners settings remain, but you have owners rights?

/cc @kristoferbaxter and @jridgewell to see what they think.

Reply:
Changed to `requestReviews: false`